extensions/amp-analytics/0.1/events.js

```diff
@@ -1468,6 +1472,29 @@ export class VisibilityTracker extends EventTracker {
       );
     }
 
+    const getUnlistenPromiseForSelector = (selector, selectionMethod) => {
```

Comment:
Should we throw user error if `selectionMethod` is defined and selector array is used?

Reply:
I introduce that in the next QuerySelectorAll PR, since it conceptually fits into that PR